recipe/meta.yaml

```diff
@@ -26,33 +26,30 @@ requirements:
   build:
     - r-base
     - r-base64enc
-    - r-catools
     - r-evaluate >=0.8
     - r-htmltools >=0.3.5
     - r-jsonlite
     - r-knitr >=1.14
+    - r-mime
     - r-rprojroot
     - r-stringr >=1.2.0
     - r-yaml >=2.1.5
     - xorg-libxrender 0.9.*
-    - xorg-libsm 1.2.*
-    - xorg-libxext 1.3.*
+    - pandoc >=2.0,<3.0
+    
   run:
-    - pandoc >=1.12.3
     - r-base
     - r-base64enc
-    - r-catools
     - r-evaluate >=0.8
     - r-htmltools >=0.3.5
     - r-jsonlite
     - r-knitr >=1.14
+    - r-mime
     - r-rprojroot
     - r-stringr >=1.2.0
     - r-yaml >=2.1.5
     - xorg-libxrender 0.9.*
-    - xorg-libsm 1.2.* 
-    - xorg-libxext 1.3.*
-    - pandoc <2.0
+    - pandoc >=2.0,<3.0
```

Comment:
This is restrictive. rmarkdown can now work with both Pandoc 1.\* and 2.\*. If another dependency were to need Pandoc 1.\*, r-rmarkdown could handle that. @bgruening Thoughts?

Reply:
I was not sure about the backwards combat of rmarkdown. If you are sure that this will work, feel free to remove the lower bond. The upper bound is also not strictly needed, but given our history here I thought this was a good idea.

Reply:
@bgruening Yep, I'm sure this will work. I've been using rmarkdown 1.8 and Pandoc 1.19.2 all morning:

```
> packageVersion("rmarkdown")
[1] ‘1.8’
> rmarkdown::pandoc_version()
[1] ‘1.19.2.1’
```

And I totally agree on the upper bound!

@bsennblad Could you please change the Pandoc run-time dependency to be `pandoc <3.0`?

Reply:
@bsennblad @bgruening Amending my previous statement. I forgot that there is a minimal version listed in the [DESCRIPTION](https://cran.r-project.org/web/packages/rmarkdown/DESCRIPTION). It's not of much consequence in this case because there are no conda packages with this low of a version, but it's better to be explicit. Let's have the Pandoc dependency be:

```
    - pandoc >=1.12.3,<3.0
```